templates/magazine/_list.html.twig

```diff
@@ -14,7 +14,9 @@
                             {% if magazine.icon %}
                                 <figure>
                                     <img width="32" height="32"
+                                         class="{{ magazine.isAdult ? 'image-adult' : '' }}"
                                          src="{{ asset(magazine.icon.filePath) | imagine_filter('avatar_thumb') }}"
+                                         {% if magazine.isAdult %}data-controller="thumb" data-action="mouseover->thumb#adult_image_hover mouseout->thumb#adult_image_hover_out"{% endif %}
```

Comment:
I was trying to track down where this is used. I might be missing it, but it looks like the magazine list page has table view and card view. I do not see a "columns" view, so I'm not sure this `elseif` gets called,  unless I'm on the wrong page

Reply:
Looks like you can still route to it manually so I went ahead and made it work and fixed the alt overflow

![image](https://github.com/MbinOrg/mbin/assets/146029455/9d6d1886-0d5a-4668-afc8-046df9ced585)